packages/nextjs/tailwind.config.js

```diff
@@ -3,6 +3,18 @@ module.exports = {
   content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}", "./utils/**/*.{js,ts,jsx,tsx}"],
   plugins: [require("daisyui")],
   darkTheme: "scaffoldEthDark",
+  theme: {
+    extend: {
+      backgroundImage: {
+        gradient: "url('/assets/gradient-bg.png')",
+      },
+      transitionDuration: {
+        2000: "2000ms",
+        3000: "3000ms",
+      },
+    },
+  },
```

Comment:
Since we are only using this in ExampleUI, I feel this pollutes the config file a bit.

Let's use this values directly on TW. `bg-[url('')]` && `duration-[2000ms]`